spdk.spec

```diff
@@ -1,15 +1,23 @@
 # doesn't seem to work on sles 12.3: #{!?make_build:#define make_build #{__make} #{?_smp_mflags}}
 # so...
-%if 0%{?suse_version} <= 1320
+%if (0%{?suse_version} <= 1320)
 %define make_build  %{__make} %{?_smp_mflags}
 %endif
 
+%if (0%{?suse_version} > 0)
+%global __debug_package 1
+%global _debuginfo_subpackages 0
+#%debug_package
+%endif
+
```

Comment:
This might need to be moved down to just before `%prep`.  Don't normally need to add this stuff for distros other than SUSE so I tend to forget the details.  But looking the mercury.spec, that's what it's doing.

Reply:
done